indexing.sql

```diff
@@ -0,0 +1,59 @@
+SELECT DISTINCT make_title
+FROM car_models
+WHERE make_code = 'LAM';
+-- 22.135ms
+
+SELECT DISTINCT model_title
+FROM car_models
+WHERE make_code = 'NISSAN' and model_code = 'GT-R';
+-- 13.503ms
+
+SELECT make_code, model_code, model_title, year
+FROM car_models
+WHERE make_code = 'LAM';
+-- 18.601ms
+
+SELECT *
+FROM car_models
+WHERE year BETWEEN 2010 and 2015;
+-- 191.071ms
+
+SELECT *
+FROM car_models
+WHERE year = 2010;
+-- 48.563ms
+
+CREATE INDEX codeIndex
+ON car_models (make_code, model_code);
+-- 374.491ms
+
+CREATE INDEX titleIndex
```

Comment:
You don't need this one at all since `make_title` and `model_title` aren't in any of the WHERE clauses.